evm_arithmetization/src/generation/state.rs

```diff
@@ -448,47 +451,42 @@ impl<F: Field> State<F> for GenerationState<F> {
         false
     }
 
-    /// Increments the `gas_used` register by a value `n`.
     fn incr_gas(&mut self, n: u64) {
         self.registers.gas_used += n;
     }
 
-    /// Increments the `program_counter` register by a value `n`.
     fn incr_pc(&mut self, n: usize) {
         self.registers.program_counter += n;
     }
 
-    /// Returns a `State`'s registers.
     fn get_registers(&self) -> RegistersState {
         self.registers
     }
 
-    /// Returns a `State`'s mutable registers.
     fn get_mut_registers(&mut self) -> &mut RegistersState {
         &mut self.registers
     }
 
-    /// Returns the value stored at address `address` in a `State`.
     fn get_from_memory(&mut self, address: MemoryAddress) -> U256 {
         self.memory.get_with_init(address)
     }
 
-    /// Returns a mutable `GenerationState` from a `State`.
+    fn get_generation_state(&self) -> &GenerationState<F> {
+        self
+    }
+
     fn get_mut_generation_state(&mut self) -> &mut GenerationState<F> {
         self
     }
 
-    /// Returns the value of a `State`'s clock.
     fn get_clock(&self) -> usize {
         self.traces.clock()
     }
 
-    /// Rolls back a `State`.
```

Comment:
All these methods are from the `State` trait, so re-documenting here is useless.